cli/file_fetcher.rs

```diff
@@ -426,10 +426,10 @@ impl SourceFileFetcher {
       return futures::future::err(e.into()).boxed_local();
     }
 
-    let is_blacklisted =
-      check_cache_blacklist(module_url, self.cache_blacklist.as_ref());
+    let is_blocklisted =
+      check_cache_blocklist(module_url, self.cache_blocklist.as_ref());
```

Comment:
Maybe just is_blocked ?

Reply:
Sure - just did a quick `sed`. I'll go through and clean these up.